### ISSUE TITLE
Spanish transformers

### ISSUE THREAD

**Issue by claverru** [explosion/spacy-transformers#125]
Hello,

Is there any chance that you could implement a wrap around @dccuchile 's BERT version (https://github.com/dccuchile/beto)?

Thank you.

**Comment by adrianeboyd** [comment]
We just added a few modifications that make it easier to initialize models from https://huggingface.co/models, which includes this model.

If you install `spacy-transformers` from source (one method: `pip install https://github.com/explosion/spacy-transformers/archive/master.zip` -- this should also include an upgrade to a newer version of `transformers`), you should be able to create a model like this:

```
python examples/init_model.py -n dccuchile/bert-base-spanish-wwm-cased -l es es_bertbasespanishwwmcased_lg
```

**Comment by CarlosGOspino** [comment]
Hi. I have attempted to replicate this method but when I try to install the library both on a local machine or on a Colab notebook I get this error:
Collecting https://github.com/explosion/spacy-transformers/archive/master.zip
  Using cached https://github.com/explosion/spacy-transformers/archive/master.zip
ERROR: Could not find a version that satisfies the requirement spacy<2.4.0,>=2.3.0 (from spacy-transformers==0.6.0) (from versions: 0.31, 0.32, 0.33, 0.40, 0.51, 0.52, 0.60, 0.61, 0.62, 0.63, 0.64, 0.65, 0.67, 0.68, 0.70, 0.80, 0.81, 0.82, 0.83, 0.84, 0.85, 0.86, 0.87, 0.88, 0.89, 0.90, 0.91, 0.92, 0.93, 0.94, 0.95, 0.97, 0.98, 0.99, 0.100.0, 0.100.1, 0.100.2, 0.100.3, 0.100.4, 0.100.5, 0.100.6, 0.100.7, 0.101.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.1.0, 1.1.1, 1.1.2, 1.2.0, 1.3.0, 1.4.0, 1.5.0, 1.5.1, 1.6.0, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.7.5, 1.8.0, 1.8.1, 1.8.2, 1.9.0, 1.10.0, 1.10.1, 2.0.0, 2.0.1.dev0, 2.0.1, 2.0.2.dev0, 2.0.2, 2.0.3.dev0, 2.0.3, 2.0.4.dev0, 2.0.4, 2.0.5.dev0, 2.0.5, 2.0.6.dev0, 2.0.6, 2.0.7, 2.0.8, 2.0.9, 2.0.10.dev0, 2.0.10, 2.0.11.dev0, 2.0.11, 2.0.12.dev0, 2.0.12.dev1, 2.0.12, 2.0.13.dev0, 2.0.13.dev1, 2.0.13.dev2, 2.0.13.dev4, 2.0.13, 2.0.14.dev0, 2.0.14.dev1, 2.0.15, 2.0.16.dev0, 2.0.16, 2.0.17.dev0, 2.0.17.dev1, 2.0.17, 2.0.18.dev0, 2.0.18.dev1, 2.0.18, 2.1.0, 2.1.1.dev0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.1.5, 2.1.6, 2.1.7.dev0, 2.1.7, 2.1.8, 2.1.9, 2.2.0.dev10, 2.2.0.dev11, 2.2.0.dev13, 2.2.0.dev15, 2.2.0.dev17, 2.2.0.dev18, 2.2.0.dev19, 2.2.0, 2.2.1, 2.2.2.dev0, 2.2.2.dev3, 2.2.2.dev4, 2.2.2, 2.2.3.dev0, 2.2.3, 2.2.4, 3.0.0.dev8)
ERROR: No matching distribution found for spacy<2.4.0,>=2.3.0 (from spacy-transformers==0.6.0)

It seems to me that the the Spacy version definition is noncompliable. I did not find spacy versions that satisfy the requirement. Best. CO
